embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/Embrace.java

```diff
@@ -576,8 +569,6 @@
         if (verifyNonNullParameters("trackWebViewPerformance", tag, consoleMessage)) {
             if (consoleMessage.message() != null) {
                 trackWebViewPerformance(tag, consoleMessage.message());
-            } else {
```

Comment:
This never gets hit as the null message cases will not make it pass the null checks.